packages/react/src/components/TableOfContents/TableOfContents.js

```diff
@@ -109,6 +111,16 @@ const TableOfContents = ({ menuItems, children, menuLabel }) => {
     setSelectedTitle(title);
   };
 
+  /**
+   * Add the dark theme class if it's selected
+   *
+   * @param {string} selected selected theme string, g100 if dark empty if light
+   * @returns {string} get dark theme class if it's selected
+   */
+  const getSelectedTheme = selected => {
+    if (selected == 'g100') return `${prefix}--tableofcontents--g100`;
```

Comment:
Can we just copy reuse the same logic that we have for other patterns [here](https://github.com/carbon-design-system/ibm-dotcom-library/blob/287fde438bee8e1f3d973fea05e366399b7216d9/packages/patterns-react/src/patterns/UseCases/UseCases.js#L89)

So if other themes need to be added, we don't have to have another conditional for it